### ISSUE TITLE
summarize in bullet points

### ISSUE THREAD

**Issue by Fukislim** [josStorer/chatGPTBox#377]
Hi, I like the summorize function but sometimes it's quite a big continues sheet of information. I think a setting where you can change your rpeference for' summorize in a sheet' or 'summorize in bullet points' would be nice.

Like I said maybe just a setting that changes the prompt style?

**Comment by josStorer** [comment]
I'm busy with other projects recently, about that PR, I think I may need to give users a switch option, because the prompt is too long and may lead to high fees, and it is not friendly enough for multilingual users, which may lead to always answering in English

**Comment by alwinraju** [comment]
Sounds good, does the current or previous versions of prompts cater for different languages?

**Comment by josStorer** [comment]
Yes, in the previous versions, replies of the corresponding language can be generated relatively stably, but recently due to the update of the prompt, the stability has dropped. For short videos, replies of the corresponding language can still be generated stably
